README.md

```diff
@@ -106,19 +75,35 @@ const MyPlaidComponent = () => {
 
 `PlaidLink` wraps the view you provide as a child with a `Pressable` component and intercepts the `onPress` event.
 
-### OAuth requirements
+#### OAuth requirements
+
+:warning: All integrations must migrate to version 9.0.0 or later of the React Native SDK (requires version 4.1.0 or later of the iOS LinkKit SDK) by September 30, 2023, to maintain support for Chase OAuth on iOS. 
+
+##### Android OAuth Requirements
+
+###### Register your app id
+1. Log into your [Plaid Dashboard](https://dashboard.plaid.com/team/api) and navigate to the API page under the Developers tab.
```

Comment:
one more nit: I think this link should actually go to https://dashboard.plaid.com/developers/api